src/components/RegisterStack.tsx

```diff
@@ -114,7 +114,7 @@ export const EmailForm: FC<EmailFormProps> = ({ navigation }) => {
         value={email || ""}
         onChangeText={setEmail}
         validity={validity}
-        long
+        length={7}
```

Comment:
does length add some validation checks here? If you have a long name are you screwed lol?

Reply:
not sure lol